platforms/jvm/scala/src/integTest/groovy/org/gradle/scala/compile/UpToDateScalaCompileIntegrationTest.groovy

```diff
@@ -56,9 +56,9 @@ class UpToDateScalaCompileIntegrationTest extends AbstractIntegrationSpec implem
         executedAndNotSkipped ':compileScala'
 
         where:
-        newScalaVersion | newZincVersion
-        '2.11.12'       | '1.6.0'
-        '2.12.6'        | '1.6.1'
+        newScalaVersion  | newZincVersion
+        '2.11.12'        | '1.6.0'
+        '2.12.18'        | '1.6.1'
```

Comment:
These have to be different versions from the defaults below so that we're testing the up-to-date path when upgrading from one version of Scala to another

Reply:
I think this is working as intended? I don't see how this violates that.

Reply:
🤦 I can't read.  This looks right, I thought the diff was on the `2.11.12` line.